src/dymaptic.GeoBlazor.Core/Components/Layers/LayerObject.cs

```diff
@@ -36,10 +36,14 @@ public virtual async Task SetSymbol(Symbol symbol)
     {
         Symbol = symbol;
 
-        if (JsGraphicReference is not null)
+        if (JsModule is not null)
         {
-            await JsGraphicReference.InvokeVoidAsync("setSymbol", 
-                Symbol.ToSerializationRecord());
+            await JsModule.InvokeVoidAsync("setGraphicSymbol", 
```

Comment:
is there any way to have a more concrete reference to these calls? Seems possible to mis-type them, or lose track of how many there are

Reply:
1. The called methods are in TypeScript, so I haven't discovered a way to verify them. We would probably have to write our own build-time method checker.
2. JsModule is normally a `CascadingParameter`, which means the one on `MapView` gets shared with all child components. However, since `Graphics` can regularly be added via code instead of razor components, I had to do a bit of juggling to make sure it was initialized in that class. For anything set in markup, it will cascade.